packages/docusaurus-theme-search-algolia/src/theme/SearchBar/algolia.css

```diff
@@ -518,21 +310,19 @@
+    center no-repeat;
   background-size: 100%;
   overflow: hidden;
   text-indent: -9000px;
   width: 110px;
   height: 100%;
   display: block;
-  margin-left: auto;
-  margin-right: 5px;
+  margin: 0 5px 0 auto;
```

Comment:
This sets the vertical margin to 0, which is a bit different. It'll be different in case there's another selector which changes the vertical margin. But should be fine.